index.html

```diff
@@ -2855,6 +2858,87 @@ <h2 class="icon-title">
           </div>
         </section>
       </section>
+      <section>
+        <h2>
+          Single color icons
+        </h2>
+        <p>
+          Some platforms enforce that icons be displayed with a <dfn>single
+          color</dfn>, where only the transparency of the icon can be
+          controlled. As web applications need to across multiple
+          platforms, it is possible to indicate that an icon can have a
+          user-agent-specified color applied by adding the <a>single-color</a>
+          purpose. This allows the platform to ensure that the icon looks well
+          integrated with the platform, and even apply different colors and
+          padding in different places throughout the platform.
+        </p>
+        <p>
+          When processing a <a>single-color</a> icon, the user agent MUST NOT
+          use the red component, green component, or blue component of a pixel.
+          The user agent SHOULD display each pixel with its original alpha
+          value, but with a red, green, and blue value of the user agent's
+          choosing. It is RECOMMENDED that the user agent use the same color
+          value for all pixels.
+        </p>
+        <p class="note">
+          Designers of <a>single-color</a> icons are recommended to set all
```

Comment:
This recommendation doesn't help in any way - why not use "could" or so instead of "recommended"